schemactl/validate.py

```diff
@@ -0,0 +1,65 @@
+import os
+import textwrap
+import humps
+import yaml
+import schema
+
+from os import path
+
+from schema import SchemaEntry
+
+def r2mm_dir() -> str:
```

Comment:
Doesn't return a string and should probably use the implementation provided in `schemactl/schema.py` (or maybe utils?) anyway.